libs/Geometry3D/test/CLinkedVector3DTests.cpp

```diff
@@ -95,6 +95,54 @@ namespace Geometry3DCLinkedVector3DTests {
         EXPECT_DOUBLE_EQ(expected_len, linked_vector.getLen());
     }
 
+    TEST_F(CLinkedVector3DFixture, CLinkedVector3D_getPointBegin_Test) {
```

Comment:
Compare addresses, not point coordinate values
EXPECT_EQ(&point_begin, &point_begin_copy)

if not const reference not allowed here, const_cast can be used
EXPECT_EQ(&point_begin, &(const_cast<CPoint3D&>point_begin_copy));

Reply:
Complete